modules/core/src/main/java/org/apache/ignite/spi/discovery/tcp/ServerImpl.java

```diff
@@ -7282,17 +7286,29 @@ private InetSocketAddress checkConnection(List<InetSocketAddress> addrs, int tim
                         for (int i = 0; i < addrsToCheck; ++i) {
                             InetSocketAddress addr = addrs.get(addrIdx.getAndIncrement());
 
+                            String logMsg = "Checking connection to node: [nodeId=" + node.id() + ", address=" + addr + ", result=";
```

Comment:
Suggestion: let's remove any '[' and ']'. Just realized we do not put in the log any composed object here. Same below. WDYT?